store/localstore/goleveldb/goleveldb.go

```diff
@@ -63,6 +63,24 @@ func (d *db) Seek(startKey []byte) ([]byte, []byte, error) {
 	return iter.Key(), iter.Value(), nil
 }
 
+func (d *db) MultiSeek(keys [][]byte) []*engine.MSeekResult {
+	iter := d.DB.NewIterator(&util.Range{Start: []byte{0x0}}, nil)
```

Comment:
If we sort the keys and using the minimum key as the startKey, can this be more effective?